econml/sklearn_extensions/federated_learning.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) PyWhy contributors. All rights reserved.

```

Comment:
I don't think `sklearn_extensions` is the best place to put this; I think `econml.federated_learning.FederatedEstimator` is probably a better location and name - so move this file up one directory and add `federated_learning` to the list in `econml\__init__.py`.

Reply:
Done.